WordPress/Classes/ViewRelated/Blog/BlogSelectorViewController.m

```diff
@@ -378,4 +498,49 @@ - (NSArray *)sortDescriptorsWithAccountKeyPath
     return descriptors;
 }
 
+#pragma mark - UISearchController
+
+- (void)updateSearchResultsForSearchController:(UISearchController *)searchController
+{
+    self.resultsController.fetchRequest.predicate = [self fetchRequestPredicate];
+
+    NSError *error = nil;
+    if (![self.resultsController performFetch:&error]) {
+        DDLogError(@"Couldn't fetch sites: %@", [error localizedDescription]);
+    }
+
+    [self.tableView reloadData];
+}
+
+// Improves the appearance of the arrow when displayed in a popover, by matching
+// its color to the topmost view in the scrollview
+- (void)scrollViewDidScroll:(UIScrollView *)scrollView
```

Comment:
This is super neat, but it does break a bit within the rect of the search field's white bg color.

![screen shot 2016-07-18 at 1 29 13 pm](https://cloud.githubusercontent.com/assets/1873422/16925849/c3a742f0-4ceb-11e6-9d52-9f2d4d3f4023.png)